src/components/actividades/ListadoActividades.js

```diff
@@ -86,6 +94,13 @@ export default function ListadoActividades() {
                   <TableCell>{actividad.nombre}</TableCell>
                   <TableCell>{actividad.Espacio.nombre}</TableCell>
                   <TableCell>{actividad.responsable}</TableCell>
+                  <TableCell>
+                    {actividad.activa ? (
+                      <FiberManualRecordIcon className={classes.activa} />
```

Comment:
Perfecto, solo otra forma de hacerlo:


```suggestion
                      <FiberManualRecordIcon className={actividad.activa ? classes.activa : classes.inactiva} />
```

Reply:
:clap: por deshardcodear los colores.